docs/setup/gke/getting-started-gke.md

```diff
@@ -151,9 +151,10 @@ nuctl deploy helloworld \
     --http-trigger-service-type nodePort \
     --registry <URL>
 ```
->**Note:** The command above exposes the function externally using a `nodePort`. This is done for demonstration
-> purposes only. Please read more about [exposing your function](/docs/tasks/deploying-functions.md#exposing-a-function)
-> for more information.
+
+> **Note:** The command in the previous code snippet exposes the function externally using a `nodePort`.
+> This is done for demonstration purposes only.
+> For more information about exposing your function, see [Deploying Functions](/docs/tasks/deploying-functions.md#exposing-a-function).
```

Comment:
```suggestion
> For more information about exposing your function, see [Exposing your function](/docs/tasks/deploying-functions.md#exposing-a-function).
```
Refer to the specific subsection

Reply:
DONE — changed to "Exposing a function", which is the title of the referenced section (commit b293817)